### PR TITLE
CI: use chrome for testing instead of chromium

### PR DESCRIPTION
**Pull request by thiagowfx**
Update @puppeteer/browsers to 1.0.0 given
https://github.com/puppeteer/puppeteer/pull/10108 is landed.

### CODE REVIEW

**Comment by thiagowfx** [comment]
@OrKoN there's an error on "puppeteer launch" when using CFT, that only happens in E2E tests. Let me know if you have time to investigate.

**Comment by thiagowfx** [comment]
The second commit did not help, so it really seems like a puppeteer issue.

```
Error: Failed to launch the browser process! spawn /Users/runner/work/chromium-bidi/chromium-bidi/chrome/mac-113.0.5672.63/chrome-mac-x64/Google ENOENT
```

...it seems "Google" was truncated. Could it be a whitespace issue?

**Comment by thiagowfx** [comment]
The issue was: `cut -f 2`. It should have been: `cut -f 2-`.

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -26,8 +26,8 @@ jobs:
       - uses: google/wireit@setup-github-actions-caching/v1
       - name: Install and build npm dependencies
         run: npm ci
-      - name: Install Chromium
-        run: echo "chromium_binary=$(npx @puppeteer/browsers install chromium@latest | cut -f 2 -d' ')" >> $GITHUB_ENV
+      - name: Install Chrome
+        run: echo "chrome_binary=$(npx @puppeteer/browsers install chrome@latest | cut -f 2- -d' ')" >> $GITHUB_ENV
```
Comment (OrKoN) [inline .github/workflows/e2e.yml:30]: Nice! The binary on Mac might contain spaces.

**Review by OrKoN — APPROVED** [review]

**Pull request merged**